tests/components/glances/test_sensor.py

```diff
@@ -19,30 +19,36 @@ async def test_sensor_states(hass: HomeAssistant) -> None:
-    if state := hass.states.get("sensor.0_0_0_0_memory_use_percent"):
-        assert state.state == HA_SENSOR_DATA["mem"]["memory_use_percent"]
-    if state := hass.states.get("sensor.0_0_0_0_docker_active"):
-        assert state.state == HA_SENSOR_DATA["docker"]["docker_active"]
-    if state := hass.states.get("sensor.0_0_0_0_docker_cpu_use"):
-        assert state.state == HA_SENSOR_DATA["docker"]["docker_cpu_use"]
-    if state := hass.states.get("sensor.0_0_0_0_docker_memory_use"):
-        assert state.state == HA_SENSOR_DATA["docker"]["docker_memory_use"]
-    if state := hass.states.get("sensor.0_0_0_0_md3_available"):
-        assert state.state == HA_SENSOR_DATA["raid"]["md3"]["available"]
-    if state := hass.states.get("sensor.0_0_0_0_md3_used"):
-        assert state.state == HA_SENSOR_DATA["raid"]["md3"]["used"]
-    if state := hass.states.get("sensor.0_0_0_0_md1_available"):
-        assert state.state == HA_SENSOR_DATA["raid"]["md1"]["available"]
-    if state := hass.states.get("sensor.0_0_0_0_md1_used"):
-        assert state.state == HA_SENSOR_DATA["raid"]["md1"]["used"]
+    if state := hass.states.get("sensor.0_0_0_0_ssl_used"):
+        assert state.state == str(HA_SENSOR_DATA["fs"]["/ssl"]["disk_use"])
+    if state := hass.states.get("sensor.0_0_0_0_cpu_thermal_1_temperature"):
+        assert state.state == str(
+            HA_SENSOR_DATA["sensors"]["cpu_thermal 1"]["temperature_core"]
+        )
+    if state := hass.states.get("sensor.0_0_0_0_err_temp_temperature"):
+        assert state.state == str(
+            HA_SENSOR_DATA["sensors"]["err_temp"]["temperature_hdd"]
+        )
+    if state := hass.states.get("sensor.0_0_0_0_na_temp_temperature"):
+        assert state.state == str(
+            HA_SENSOR_DATA["sensors"]["na_temp"]["temperature_hdd"]
+        )
+    if state := hass.states.get("sensor.0_0_0_0_ram_used_percent"):
+        assert state.state == str(HA_SENSOR_DATA["mem"]["memory_use_percent"])
+    if state := hass.states.get("sensor.0_0_0_0_containers_active"):
+        assert state.state == str(HA_SENSOR_DATA["docker"]["docker_active"])
+    if state := hass.states.get("sensor.0_0_0_0_containers_cpu_used"):
+        assert state.state == str(HA_SENSOR_DATA["docker"]["docker_cpu_use"])
+    if state := hass.states.get("sensor.0_0_0_0_containers_ram_used"):
+        assert state.state == str(HA_SENSOR_DATA["docker"]["docker_memory_use"])
+    if state := hass.states.get("sensor.0_0_0_0_md3_raid_available"):
+        assert state.state == str(HA_SENSOR_DATA["raid"]["md3"]["available"])
+    if state := hass.states.get("sensor.0_0_0_0_md3_raid_used"):
+        assert state.state == str(HA_SENSOR_DATA["raid"]["md3"]["used"])
+    if state := hass.states.get("sensor.0_0_0_0_md1_raid_available"):
+        assert state.state == str(HA_SENSOR_DATA["raid"]["md1"]["available"])
+    if state := hass.states.get("sensor.0_0_0_0_md1_raid_used"):
```

Comment:
Do we need conditionals in this test at all? Why not remove the if and make them all like:
```
assert hass.states.get("sensor.0_0_0_0_md1_raid_used").state == str(HA_SENSOR_DATA["raid"]["md1"]["used"])
```
(Am on mobile, beware typos)

Reply:
We dislike conditional tests, so I would also prefer to see it like you described